VultisigApp/VultisigApp/View Models/FunctionCallVerifyViewModel.swift

```diff
@@ -71,6 +137,41 @@ class FunctionCallVerifyViewModel: ObservableObject {
         return keysignPayload
     }
     
+    private func getInboundChainName(for chain: Chain) -> String {
```

Comment:
again this function is duplicated https://github.com/vultisig/vultisig-ios/pull/2550/files#diff-f6eced95153c2163c9c9eaface3e7c57b6fb201a37224d89e4ff2d10f9c3b0c3R344